rosco-core/src/test/java/com/netflix/spinnaker/rosco/providers/aws/AWSStartupTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2024 OpsMx, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.rosco.providers.aws;
+
+import com.netflix.spinnaker.rosco.Main;
+import com.netflix.spinnaker.rosco.providers.aws.config.RoscoAWSConfiguration;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = {Main.class})
+@TestPropertySource(
+    properties = {
+      "spring.application.name=rosco",
+      "aws.enabled=true",
+      "rosco.config-dir=/some/path",
+      "bakeAccount=xyz"
+    })
+public class AWSStartupTest {
+
+  @Autowired AWSBakeHandler awsBakeHandler;
+
+  @Autowired RoscoAWSConfiguration.AWSBakeryDefaults awsBakeryDefaults;
+
+  @Test
+  public void startupTest() {
+    assert awsBakeHandler
```

Comment:
Can you explain why this is a good startup test?  Also, I'd rather see a more test-y assertion, like:
```suggestion
   assertThat(awsBakeHandler.getMaskedPackerParameters().toString()).isEqualToIgnoringCase("[aws_access_key, aws_secret_key]");
 ```
or even better:
```
assertThat(awsBakeHandler.getMaskedPackerParameters()).isEqualTo(List.of("aws_access_key", "aws_secret_key"));
```

Reply:
Used it as an additional marker to verify the values returned from the static class. However, it would not be required because the test is to verify the startup of rosco application with AWS provider enabled. So, removing this assert statement.